pkg/mount/README.md

```diff
@@ -0,0 +1,34 @@
+mount
+=====
+
+Mount block devices for use in Linuxkit
+
+**NOTE: At this time, block devices may only be mounted in `/var`**
```

Comment:
Because it confused me endlessly when my mount to `/boot` kept failing.
It seems that because only `/dev` and `/var` are bound by default in the `mount` container mounts outside of those locations will never propagate to the host.